plugins/cloud-gce/src/test/resources/org/elasticsearch/discovery/gce/computeMetadata/v1/instance/hostname.json

```diff
@@ -0,0 +1 @@
+www.elastic.co
```

Comment:
feels a bit weird that these have a .json extension but the content is just text...